### PR TITLE
Add css modules

### PR DESCRIPTION
**Pull request by Tareq9000**
making a style sheet

### CODE REVIEW

File: src/styles/style.module.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  text-align: center;
+  font-family: sans-serif;
+}
+.show_comments:hover{
+  color: rgb(0, 0, 255);
+  text-decoration: underline;
+  font-size: 17px;
+}
+.spinner_box{
+  position: fixed;
```
Comment (josetapadas) [inline src/styles/style.module.css:95]: A sample comment :)

**Review by josetapadas — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+.story-by{
+  color: grey;
+  font-weight: bold;
+}
+.story-time{
+  color: grey;
+  font-weight: bold;
+}
+.story-score{
+  color: grey;
```
Comment (josetapadas) [inline src/style.css:57]: the color should be red!

**Review by josetapadas — CHANGES_REQUESTED** [review]

**Comment by josetapadas** [comment]
I cannot see the index.js updated (https://github.com/Tareq9000/hacker-news/blob/master/dist/index.html) are you sure you've pushed?

File: src/styles/style.module.css (hunk 1 above)
Comment (Tareq9000) [reply src/styles/style.module.css:95]: Unbenannt

Comment (Tareq9000) [reply src/styles/style.module.css:95]: image

Comment (Tareq9000) [reply src/styles/style.module.css:95]: image

**Review by josetapadas — APPROVED** [review]
I approve this PR!

**Pull request merged**